java_console/logging/src/main/java/com/rusefi/FileLog.java

```diff
@@ -22,7 +22,7 @@ public enum FileLog {
 
     public static final String OS_VERSION = "os.version";
     public static final String DATE_PATTERN = "yyyy-MM-dd_HH_mm_ss_SSS";
-    private static final String WIKI_URL = "https://github.com/rusefi/rusefi/wiki/rusEFI-logs-folder";
+    private static final String WIKI_URL = "https://wiki.fome.tech/FOME-Console/";
```

Comment:
This URL isn't real, yet.
----
![image](https://github.com/user-attachments/assets/f65e1911-8bc3-42be-ae77-65ddbe0a1f6e)